### DISCUSSION TITLE
Add list of compositors using Louvre to the read me

### DISCUSSION THREAD

**Discussion by wiki-me** [CuarzoSoftware/Louvre#40] [Ideas]
Continuing our discussion from [here](https://www.reddit.com/r/linux/comments/1aqw077/louvre_120_release/kqizv0e/).

I suggest having a link rather then a full list like in smithay, as that would make the read me a little easier to skim , Also i don't think a wiki should be used because iirc it's can't be easily edited with a pull request, linking to some other file in markdown is good.

**Comment by ehopperdietzel** [comment]
Ok, I will create a new branch named "gallery" or similar with a README.md, the format should be something like this?

# Louvre Compositors Showcase

Welcome to the list of compositors using Louvre. If you'd like to showcase your compositor, follow the steps below:

## List of Compositors Using Louvre

* [Compositor Name 1](#) : A floating window style compositor.
* [Compositor Name 2](#) : A tiling window style compositor.
* [Compositor Name 3](#) : A 3D compositor (Work in Progress).

## Add Your Compositor

1. Click the edit (pencil) button at the top right corner.
2. Add your compositor following the format: `* [Your Compositor Name](Link to the project) : Description.`
3. Make sure to insert the line in alphabetical order among the existing compositors
4. Save your changes and create a pull request.

**Comment by ehopperdietzel** [comment]
Here is the [README.md](https://github.com/CuarzoSoftware/Louvre/blob/gallery/README.md), feel free to modify it if you wish.
